decidim-proposals/config/locales/ca.yml

```diff
@@ -120,6 +120,8 @@ ca:
           most_voted: Més votat
           random: Aleatori
           recent: Recent
+        proposal:
+          view_proposal: Veure proposta
```

Comment:
We shouldn't be adding locales directly from a PR, we should only do this via Crowdin